lib/generators/administrate/routes/templates/routes.rb.erb

```diff
@@ -1,5 +1,4 @@
 namespace :<%= namespace %> do
-<% dashboard_resources.each do |resource| %>    resources :<%= resource %>
-<%end%>
-    root to: "<%= dashboard_resources.first %>#index"
-  end
+<%= generate_resource_routes() %>
```

Comment:
Nitpick! Can you remove the brackets please? It's a style thing; you don't normally see empty brackets in Ruby code.

```suggestion
<%= generate_resource_routes %>
```

Reply:
will fix, my ruby styling will most likely get worse over time now